src/controllers/userController.js

```diff
@@ -123,8 +125,87 @@ class UserController {
       });
     }
   }
+
+  /**
+   * @description - This method is responsible for assigning roles to users
+   *
+   * @static
+   * @param {object} request - Request sent to the router
+   * @param {object} response - Response sent from the controller
+   *
+   * @returns {object} - object representing response message
+   *
+   * @memberof UserController
+   */
+  static async assignRole(request, response) {
+    try {
+      const { email, role } = request.body;
+      if (role !== 'admin' && role !== 'user') {
+        return response.status(400).json({
+          status: 'Fail',
+          message: 'Status can only be "admin" or "user"'
```

Comment:
Why not use backticks here?

Reply:
Thanks for pointing it out. It has been changed